docs/src/main/paradox/kafka.md

```diff
@@ -4,6 +4,7 @@
 
 To use the Kafka module of Akka Projections add the following dependency in your project:
 
+@@dependencies{ projectId="akka-projection-kafka" }
```

Comment:
pls revert, that's covered in my Kafka docs PR

Reply:
strange, I don't remember to have added that